tinychat/examples/tinychat/index.html

```diff
@@ -1,111 +1,168 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
Why is prettier modifying this as well?

Reply:
That is a default behavior of prettier, you can take a look a this [issue](https://github.com/prettier/prettier/issues/15096) on prettier's github.

If you wish I can change the declaration back to uppercase, however, anytime a person formats it via prettier, it is going to be in lowercase again.

Hence, a permanent solution would be to add `<!-- prettier-ignore -->` above the `<!DOCTYPE html>` declaration.